airbyte-cdk/python/airbyte_cdk/test/utils/data.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from pathlib import Path as FilePath
+
+
+def _get_resource_root_folder(execution_folder: FilePath) -> FilePath:
```

Comment:
This seems to return the unit_tests folder and not the resource one. I'm I mistaken?